boot/bootdata/txtsetup.sif

```diff
@@ -15,8 +15,7 @@ Signature = "$Windows NT$"
 _1 = %cdname%,%cdtagfile%,,""
 
 [SourceDisksNames.x86]
-;1 = %cdname%,%cdtagfilei%,,\i386
-1 = %cdname%,%cdtagfilei%,,\reactos
+1 = %cdname%,%cdtagfilei%,,\i386
```

Comment:
You might want to do the same for the amd64 section below.

Reply:
good idea but I wanted to start "small". And I need to test the x64 builds, now that we can use them (wasn't the case back in the day when this commit was first prepared.)